martha/martha_v3.js

```diff
@@ -426,37 +427,28 @@ function overlapFields(requestedFields, serviceFields) {
     return requestedFields.filter((field) => serviceFields.includes(field)).length !== 0;
 }
 
-function sendResponse(res, requestedFields, drsResponse, fileName, bondProvider, bondSA, accessUrl) {
-    const fullResponse =
-        requestedFields.length
-            ? convertToMarthaV3Response(drsResponse, fileName, bondProvider, bondSA, accessUrl)
-            : {};
-    const partialResponse = mask(fullResponse, requestedFields.join(","));
-
-    res.status(200).send(partialResponse);
-}
+function parseOptions(opt) {
```

Comment:
nit: some of these "options" are not optional, e.g. `url` and `auth`. Perhaps `parseParameters(params)` or something like that?